index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal-Portfolio-Page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <nav class="nav row">
+                <ul class="nav-logo">
+                    <li class="logo-item"><img src="img/image_geometry_1.svg" alt="logo-image" class="logo-img"></li>
+                </ul>
+                
+                <menu class="nav-menu">
+                    <button>
+                        <div class="line"></div>
+                        <div class="line space-between"></div>
+                        <div class="line"></div>
+                    </button>
+                </menu>
+            </nav>
+        </header>
+
+        <section class="headline row">
+            <div class="container">
+                <h1 class="headline-name darkblue">Binyam Y. Olango</h1>
+                <p class="headline-desc darkblue">Hello! I am a software developer! I can help you build a product, feature or website. 
+                    Take a look at my work.<br><br>If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                <button class="headline-btn">Start collaboration</button>
```

Comment:
- [ ] For great user interaction and experience; kindly implement the Hover and Pressed button styling according to the given template. 👍 

<img width="136" alt="Untitled" src="https://github.com/binyamolango/Personal-Portfolio/assets/103386183/d4303276-1415-4345-82fb-ce809d44d3d0">

Reply:
@OyePriscilla Thank you so much for the comments. I will get through them right away. Can you check the 5th comment though? I believe I have implemented the Hover and Pressed button styling. Thanks🙏